zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
```

Comment:
Another option to consider would be to have a separate fn called `pick_multiple` not sure I have a strong opinion, just think of the pros and cons.